libs/client/LightEpoch.cs

```diff
@@ -100,15 +100,15 @@ static LightEpoch()
 
             // Over-allocate to do cache-line alignment
 #if NET5_0_OR_GREATER
-            threadIndex = GC.AllocateArray<Entry>(kTableSize + 2, true);
+            threadIndex = GC.AllocateArray<Entry>(KTableSize + 2, true);
             p = (long)Unsafe.AsPointer(ref threadIndex[0]);
 #else
             threadIndex = new Entry[kTableSize + 2];
```

Comment:
This and all `#if !NET5_0_OR_GREATER` is dead code, which is why this wasn't automatically renamed. You can either just rename this or remove all `#if !NET5_0_OR_GREATER` if defs.